packages/create-plugin/templates/common/.config/webpack/BuildModeWebpackPlugin.ts

```diff
@@ -0,0 +1,29 @@
+import * as webpack from 'webpack';
+
+const PLUGIN_NAME = 'BuildModeWebpack';
+
+export class BuildModeWebpackPlugin {
+  apply(compiler: webpack.Compiler) {
+    compiler.hooks.compilation.tap(PLUGIN_NAME, (compilation) => {
+      compilation.hooks.processAssets.tap(
+        {
+          name: PLUGIN_NAME,
+          stage: webpack.Compilation.PROCESS_ASSETS_STAGE_ADDITIONS,
+        },
+        async () => {
+          const pluginJsonAsset = compilation.getAsset('plugin.json');
```

Comment:
I know this is draft but does this work with nested plugins?

Reply:
No 😱  Fixed in https://github.com/grafana/plugin-tools/pull/1281/commits/9616e88d5514f83d9f9a9967acbf993385359162